Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -201,6 +204,9 @@ private void OnExamine(EntityUid uid, SuitSensorComponent component, ExaminedEve
                 return;
         }
 
+        if (HasComp<EmpDisabledComponent>(uid))
+            msg = "suit-sensor-examine-off";
+
```

Comment:
EMP should just be setting this rather than tightly coupling 2 systems together.

If a fork wants suit sensors but no EMP suddenly they need to keep EMPs around due to it.

Reply:
Ok, I can switch back to setting the value.

Reply:
I've separated the logic out into just the EMP events, and it just sets it to Disabled on EMP and back again after it ends.